workbench-core/accounts/src/services/projectService.ts

```diff
@@ -80,4 +80,21 @@ export default class ProjectService {
 
     return Promise.resolve({ data: projectsResponse.Items as unknown as Project[] });
   }
+
+  /**
+   * Check whether a CostCenter have any projects associated with it
+   * @param costCenterId - id of CostCenter we want to check
+   * @returns Whether a CostCenter have any projects associated with it
+   */
+  public async doesCostCenterHaveProjects(costCenterId: string): Promise<boolean> {
+    const queryParams: QueryParams = {
```

Comment:
You should still add a `limit` parameter so we have bounded queries. This only matters if one is found so we can use `limit: DEFAULT_API` or `limit: 5` even.

Reply:
Good idea. I'll make the limit `1` since we just need to know whether a project exist.